koswat/configuration/io/koswat_run_settings_importer.py

```diff
@@ -74,6 +74,9 @@ def import_from(self, from_path: Path) -> KoswatRunSettings:
         _dike_selected_sections = self._import_selected_dike_section_names(
             _general_settings.analysis_section.dike_selection_txt_file
         )
+        if not _dike_selected_sections:
+            logging.error("No dike sections selected for analysis.")
+            return KoswatRunSettings()
```

Comment:
why not raising an error and stopping the whole process instead? From my understanding that's what the user wants right?